client/__tests__/e2e/puppeteerUtils.ts

```diff
@@ -60,7 +60,7 @@ export async function tryUntil(
     maxRetry = 50,
     timeoutMs = RETRY_TIMEOUT_MS,
     page,
-    silent = false,
+    silent = true,
```

Comment:
tryUntil now defaults silent=true to reduce log noise. Confirm this won’t hide useful diagnostics when debugging CI flakes.